### DISCUSSION TITLE
Unable to create a micro - not enough funcs in reserve

### DISCUSSION THREAD

**Discussion by samjheath** [deta/cloud-docs#352] [Help] [answered]
Hey, so I've used Deta once before and found it to be super seamless install and setup for a project.
I've been trying to create a new micro (have followed along with the quickstart guide to make sure I'm not doing something wrong) but keep on running into this error and no clear way around it.

```
Error: failed to create new program: not enough funcs in reserve
```

Looking at the CLI source code looks like the request to create a new program is failing on Deta side of things. Possibly some underlying aws account limitation perhaps 🤷🏻‍♂️ , just hoping to raise the issue.

**Comment by aavshr** [comment, marked as answer]
Hi @samjheath, sorry for this, a bad config update caused this issue. It should be resolved now.

**Reply by samjheath** [reply]
Awesome thanks for the quick update 👍
